src/test/bind/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+include_directories(${GLIB_INCLUDES})
+link_libraries(${GLIB_LIBRARIES})
+
 ## build the test as a dynamic executable that plugs into shadow
 add_library(shadow-plugin-test-bind SHARED test_bind.c)
```

Comment:
FYI: I think we don't want to change this to `add_shadow_exe` because the library version works well enough and we want to minimize conflicts with the phantom branch.

Reply:
Agreed. I didn't touch the targets here; just added the GLIB includes and libraries.